scripts/jenkins/cloud/ansible/roles/list_packages/tasks/list-packages.yml

```diff
@@ -0,0 +1,46 @@
+#
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+
+
+- name: Collect list of installed packages in plaintext
+  shell: |
+    rpm -qa | sort > /tmp/rpms/rpms_{{ inventory_hostname }}_{{ suffix }}.txt
+  args:
+    warn: False
+    creates: "/tmp/rpms/rpms_{{ inventory_hostname }}_{{ suffix }}.txt"
```

Comment:
Well this doesn't count with "another update after update". The whole role handles 2 states e.g. *before* and *after* update. Action "diff" should be run at the very end.

Reply:
It could be done as  snapper of installed packages and at the end just choose which states u want to compare (like before the admin install, after the admin install and after update). But this is not what description of card says.

Reply:
Ok, got it.

But whats the advantage of having the creates? I can only see this causing problems if for some reason the file already exists.

Reply:
I redone the task and no `created` is not present